### DISCUSSION TITLE
Prevent closing popover on interaction

### DISCUSSION THREAD

**Discussion by philmetzger** [adobe/react-spectrum#5841] [Q&A]
I have a simple implementation for a dropdown multi select.

```
<ComboBox
      onSelectionChange={onSelect}
      className="group flex w-full flex-col gap-1"
      aria-label={ariaLabel || label || 'Select an item'}
      menuTrigger="focus"
      items={items}
    >
      {label && (
        <Label className="mb-1">
          <Text transform="uppercase" size="xs" color="gray-500" fontWeight="bold">
            {label}
          </Text>
        </Label>
      )}
      <Group className="flex rounded-full border-2 border-gray-800 bg-gray-950 transition focus-visible:ring-2 focus-visible:ring-black">
        <Input placeholder={placeholder} className={inputClassNames} />
        <Button className="flex items-center justify-center rounded-r-lg bg-transparent px-2 text-gray-700 transition">
          <ChevronUpDownIcon className="w-7 text-gray-500" />
        </Button>
      </Group>
      <Popover className="w-[--trigger-width] overflow-auto rounded-lg border border-gray-800 bg-gray-900 shadow-lg entering:animate-in entering:fade-in exiting:animate-out exiting:fade-out">
        <ListBox>
          {(item: { id: string; name: string }) => (
            <MultiSelectItem
              key={item.id}
              id={item.id}
              textValue={item.name}
              isSelected={selected.includes(item.id) || selected.includes(item.name)}
              size={size}
            />
          )}
        </ListBox>
        {footer && footer}
      </Popover>
    </ComboBox>
```

The problem I have is when I select an item in the listBox, the popover closes. How can I configure it so depending on what i want i can keep the popover open (until i click outside) or close it after selection

**Comment by snowystinger** [comment]
Multiselection in combobox is not supported. Please see https://github.com/adobe/react-spectrum/issues/2140 for more information.
In addition, please do not put other interactive elements inside a combobox; they are not reachable via keyboard and, therefore, are not accessible.

This is both for inside individual items https://react-spectrum.adobe.com/react-aria/ComboBox.html#text-slots
and for the popover as a whole (ie footer). Focus is managed virtually and never actually leaves the input where the user is typing and can edit their current value.

Also relevant
https://github.com/adobe/react-spectrum/discussions/5489
https://github.com/adobe/react-spectrum/discussions/2162
